### PR TITLE
update requirements

### PR DESCRIPTION
**Pull request by FuhuXia**

### CODE REVIEW

File: requirements-freeze.txt [hunk 1]
```diff
@@ -1,19 +1,18 @@
 Babel==2.3.4
 Beaker==1.9.0
-bleach==3.3.0
+bleach==3.1.4
```
Comment (jbrown-xentity) [inline requirements-freeze.txt:3]: Any idea why it downgraded bleach?

**Review by jbrown-xentity — APPROVED** [review]

File: requirements-freeze.txt (hunk 1 above)
Comment (FuhuXia) [reply requirements-freeze.txt:3]: 3.3.0 was manually added in the freeze file. the auto generated freeze file will overwrite it.
We need to take another look at https://github.com/GSA/inventory-app/pull/246/files
if bleach is latest version, then we dont need a pinned version in the requirements.txt any more.

Comment (chris-macdermaid) [reply requirements-freeze.txt:3]: @FuhuXia This is the lasted version of bleach and could be set without it being pinned to 3.3.0. Although in the [CKAN master,](https://github.com/ckan/ckan/blob/master/requirements.txt#L10) bleach is pinned at 3.3.0.

Comment (FuhuXia) [reply requirements-freeze.txt]: Rebased to inventory_ckan_2.8 for the requirements.txt change and this time it stays at 3.3.0. marking resolved.

**Pull request merged**